src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String taskLabel;
```

Comment:
I like how you have added another variable called **taskLabel**.

This is actually clear while storing the data in it instead of explicitly typing [D] / [E] / [T] every time while printing the lists.